pkgs/development/libraries/graphene/default.nix

```diff
@@ -94,7 +94,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A thin layer of graphic data types";
-    homepage = "https://ebassi.github.com/graphene";
+    homepage = "https://github.com/ebassi/graphene";
```

Comment:
```suggestion
    homepage = "https://ebassi.github.io/graphene/";
```

Reply:
Generally I have a preference for the repo, as that's usually what I want when troubleshooting... :/